src/components/Footer.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+const Footer = props => (
```

Comment:
Suggestion: use object destructuring to avoid repeated word `props` 
```javascript
const {countActiveItems, showSelectedList, /* other props */} = props;
```